spring-context-support/src/main/java/org/springframework/cache/jcache/JCacheCache.java

```diff
@@ -42,15 +42,15 @@
 
 
 	/**
-	 * Create an {@link org.springframework.cache.jcache.JCacheCache} instance.
+	 * Create a {@link org.springframework.cache.jcache.JCacheCache} instance.
```

Comment:
The package name starts with an "o". Thus this is grammatically correct.

Please undo this change.

Reply:
@sbrannen "a" is intended for the rendered link in Javadoc, "JCacheCache", not "org.springframework.cache.jcache.JCacheCache". Am I missing something?

See https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/cache/jcache/JCacheCache.html

Reply:
Hmmm.... good question!

I believe we (i.e., the collective core Spring Team) were thinking that the fully qualified class name (FQCN) shows up in the generated HTML when linked via FQCN.

So, if that's not the case, that would mean there is a larger issue across the entire code base in the core Spring Framework, since we routinely use "an" instead of "a" in such cases.

Perhaps it's better to address that nuanced feature of Javadoc in a dedicated JIRA issue.

@snicoll, thoughts?

Reply:
@sbrannen thanks. Could you please raise a SPR issue for this? I got caught as well and I am wondering if that's not a new feature of javadoc or something (or I've been wrong this all time).

Reply:
Indeed! I am also wondering if this was a change to how Javadoc works in a recent release (perhaps in JDK 8?).

And well... if you've been wrong this whole time, so have I and many others. 😉 

I'll raise a JIRA issue to track this.

Reply:
Done: https://jira.spring.io/browse/SPR-17208